mulhome_scripts/k8s_circe_scheduler.py

```diff
@@ -47,8 +49,8 @@ def check_status_circe(dag,app_name):
 
     result = True
     for key, value in dag.items():
-        # print(key)
-        # print(value)
+        # logging.debug(key)
```

Comment:
Delete commented code. This will be the last time i mention this in this review. Please put attention to all the commented code I did not add a review comment to. This is a good time to do this since you've just made changes to these commented lines of code.

Reply:
We can also put this job into a completely separate pull request. To organize our thoughts.

Reply:
I tried to check but there are too many files so I missed some of them.